### PR TITLE
Highlight importance of setting up minicom pre interview

### PR DESCRIPTION
**Pull request by pratik60**
Additional changes include 

- Prettier config
- Changes to node setup to avoid watching on db.json which was causing this infinite loop. 

```
Admin listening on port 3000                   

crashing child                                 

Starting child process with 'node app'         

Changes detected: F:\minicom-main\node\db.json 

Changes detected: F:\minicom-main\node\db.json 
```

### CODE REVIEW

File: node/package.json [hunk 1]
```diff
@@ -21,6 +21,6 @@
     "npm": ">=1.4.0"
   },
   "scripts": {
-    "start": "supervisor app"
+    "start": "supervisor -RV -i ./db.json app"
```
Comment (pratik60) [inline node/package.json:24]: -RV|--restart-verbose - Logs the file(s) that caused supervisor to restart

File: README.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
 
 Welcome to Minicom! A prototype Intercom service. This is a project to help you set up for your interview later on. Inside this directory you'll find a foo website, a bar website, and servers that represent a simplified version of how Intercom works.
 
+# What do I have to do?
```
Comment (pratik60) [inline README.md:5]: @patocallaghan Do you think this is more explicit?

Comment (patocallaghan) [reply README.md:5]: Nice, I like this 👍

**Review by patocallaghan — APPROVED** [review]
🙌

**Pull request merged**